scopes/react/react-native/webpack/webpack-transformers.ts

```diff
@@ -1,10 +1,11 @@
+import { join } from 'path';
 import { WebpackConfigTransformer, WebpackConfigMutator, WebpackConfigTransformContext } from '@teambit/webpack';
 import { get, set } from 'lodash';
 import { reactNativeAlias } from './react-native-alias';
 
 const reactNativePackagesRule = {
   test: /\.(jsx?|tsx?)$/,
-  include: [/node_modules\/react-native-/],
+  include: [new RegExp(join('node_modules$', 'react-native-/'))],
```

Comment:
this might create wrong regex. maybe just replace `\/` with `[\\\/]`?

Reply:
I don't think the new RegExp is really needed.
also, maybe better to use this component - https://github.com/teambit/bit/blob/d7c9c7e36b7b8f9bd91ea4e80d991804d9ff3fa9/scopes/toolbox/path/to-windows-compatible-path/to-windows-compatible-path.ts#L7

Reply:
that component wont help - it doesnt change / to\\, just a single to a double
the problem here is that the / (or \/ as escaped /) isnt being matched to the windows path

Reply:
What would you do instead of RegExp? path.join wont produce a regular expression